scripts/spack/packages/axom/package.py

```diff
@@ -3,12 +3,12 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
-
 import os
 import socket
 from os.path import join as pjoin
 
+from spack import *
+
 
 def get_spec_path(spec, package_name, path_replacements={}, use_bin=False):
```

Comment:
This problem seems like it would affect multiple codes.

On the spack side, perhaps it should be moved higher, e.g. to the `CachedCMakePackage` class?

Reply:
Which problem? spec path or the implicit link?  the implicit link fix requires BLT.

Reply:
Good point!

Reply:
(I was thinking about the implicit link fix for blueos xl)